perl/p5-graveyard/Portfile

```diff
@@ -1706,16 +1706,6 @@ foreach branch $defaultBranches {
         replaced_by subversion-perlbindings-$replacements($branch)
     }
 
-# p5-datemanip replaced by p5-date-manip
-# remove this subport after 2019-06-11
-
-    subport p$branch-datemanip {
-       version 6.700.0
-       revision 2
-       replaced_by p$replacements($branch)-date-manip
-    }
-}
```

Comment:
it looks like this last ```}``` should not have been removed?

Reply:
You are right, I removed it by mistake. #4595